easybuild/easyconfigs/r/Racon/Racon-1.4.7-fosscuda-2019b.eb

```diff
@@ -0,0 +1,163 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'CMakeMake'
+
+name = 'Racon'
+version = '1.4.7'
+
+homepage = 'https://github.com/clara-genomics/racon-gpu'
+description = "Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/clara-genomics/racon-gpu/archive'],
+        'filename': '%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/clara-genomics/ClaraGenomicsAnalysis/archive'],
+        'filename': 'ClaraGenomicsAnalysis.zip',
+        'download_filename': '416af9f1817a4a70745b3f7cdb7418125159f75c.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/bioparser/archive'],
+        'filename': 'bioparser.zip',
+        'download_filename': '0d66534d75b319f43ce5c59fe6548eee48f8f651.zip',
+    },
+    {
+        'source_urls': ['https://github.com/martinsos/edlib/archive'],
+        'filename': 'edlib.zip',
+        'download_filename': 'ee36875e944b63abf6f9b9e14c010b445529427d.zip',
+    },
+    {
+        'source_urls': ['https://github.com/google/googletest/archive'],
+        'filename': 'googletest.zip',
+        'download_filename': 'ea31cb15f0c2ab9f5f5b18e82311eb522989d747.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/logger/archive'],
+        'filename': 'logger.zip',
+        'download_filename': 'b88c8ee8ba59fd07b40c022957b7bdce1a5f8502.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/rampler/archive'],
+        'filename': 'rampler.zip',
+        'download_filename': 'a6dec0140878ba81da90387c77740756fefa4920.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/spoa/archive'],
+        'filename': 'spoa.zip',
+        'download_filename': '11219575bdcb356c57fb32f54cf23b8da2abc4f0.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/thread_pool/archive'],
+        'filename': 'thread_pool.zip',
+        'download_filename': '048da9485390ee8b8a7ecedccc598a06ca4429e2.zip',
+    },
+    {
+        'source_urls': ['https://github.com/madler/zlib/archive'],
+        'filename': 'zlib.zip',
+        'download_filename': 'cacf7f1d4e3d44d871b605da3b647f07d718623f.zip',
```

Comment:
Do we really need a custom `zlb`, we can't use a proper `zlib` dependency?

Reply:
Judging from Cmake in bioparser, which requires the zlib I think the other solution would be creating patch file to change CMakeLists.txt https://github.com/rvaser/bioparser/blob/0d66534d75b319f43ce5c59fe6548eee48f8f651/CMakeLists.txt#L17-L20
would it be worth it the time?

Reply:
I'd say yes, `zlib` is very common, it doesn't make sense to build a custom version here...

What happens if you just don't include `vendor/zlib` at all, and just add `zlib` as a dep?

The compiler should be able to find the `zlib` header files, since we update `$CPATH`...